src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -1504,6 +1504,422 @@ public static void Main()
             );
         }
 
+        public void WithExpr_DefiniteAssignment_01()
```

Comment:
>WithExpr_DefiniteAssignment_01 [](start = 20, length = 30)

It's cool that CI caught missing [Fact] here.
Note: Andy's latest PR generates Clone method on records.

Reply:
welp! Thanks, CI

Reply:
I think this might have been from a bad merge

Reply:
Yeah this particular analyzer diagnostic from the xunit assembly has saved me many times.